.github/workflows/ci-build.yml

```diff
@@ -42,6 +47,44 @@ jobs:
         run: yarn build
       - name: Build extension ROSE Wallet
         run: yarn build:ext
+      - name: Sync Capacitor for Android
+        run: yarn cap sync android
+      - name: Accept SDK licenses
+        run: yes | $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --licenses
+      - name: Install SDK components
+        run: |
+          $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager "platform-tools" "platforms;android-31" "build-tools;31.0.0"
```

Comment:
Can the app be build with older SDK version(Since we are targeting 34)?

Reply:
ah thanks need to sync after https://github.com/oasisprotocol/oasis-wallet-web/pull/1957